src/quo2/components/info/network_breakdown.cljs

```diff
@@ -0,0 +1,77 @@
+(ns quo2.components.info.network-breakdown
+  (:require [quo2.components.icon :as icons]
+            [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [status-im.i18n.i18n :as i18n]
+            [clojure.string :as string]))
+
+(defn network-breakdown
+  [{:keys [top-value network-conversions]}]
+  [rn/view {:style {:background-color (colors/theme-colors
+                                       colors/white
+                                       colors/neutral-95)
+                    :padding-horizontal 40
+                    :padding-top 24
+                    :border-radius 16
+                    :height 126
+                    :width 390
+                    :overflow :hidden}}
+   [rn/view {:style {:border-bottom-width 1
+                     :border-bottom-color (colors/theme-colors
+                                           colors/neutral-20
+                                           colors/neutral-70)
+                     :padding-vertical    6}}
+    [text/text {:weight :medium
+                :style  {:color     (colors/theme-colors
+                                     colors/black
+                                     colors/white)
+                         :font-size 19}}
+     (str top-value)]]
```

Comment:
As far as I have tested, I think we never need to worry about converting something to a string when using components such as ReactNative Text, as this is the default behavior.

Reply:
I used it just to handle the case of `nil` to make it an empty string.